es5.md

```diff
@@ -531,7 +531,7 @@ A unary operator token.
 interface UpdateExpression <: Expression {
     type: "UpdateExpression";
     operator: UpdateOperator;
-    argument: Expression;
+    argument: Pattern | CallExpression;
```

Comment:
@michaelficarra that would be a good note to add, but I observe that this is already the case: e.g. in [es2015](https://github.com/estree/estree/blob/master/es2015.md) the LHS of an AssignmentExpression must be a Pattern, but this is only required because of an [early error](https://tc39.github.io/ecma262/#sec-assignment-operators-static-semantics-early-errors).